cctrl/auth.py

```diff
@@ -119,6 +220,15 @@ def get_password(create=False):
     return password
 
 
+def get_password_env(settings):
+    try:
+        password = os.environ.pop(settings.login_creds['pwd'])
```

Comment:
afaik it doesn't actually remove things from the environment, just from its local copy of the environment. but I can switch to `os.getenv` anyway.